pkgs/applications/blockchains/bisq-desktop/default.nix

```
@@ -60,16 +47,26 @@ stdenv.mkDerivation rec {
       name = "Bisq";
       exec = "bisq-desktop";
       icon = "bisq";
-      desktopName = "Bisq";
+      desktopName = "Bisq ${version}";
       genericName = "Decentralized bitcoin exchange";
-      categories = "Network;Utility;";
+      categories = "Network;P2P;";
     })
   ];
 
   unpackPhase = ''
     dpkg -x $src .
   '';
 
+  buildPhase = ''
+    # Replace the embedded Tor binary (which is in a Tar achive)
+    # with one from Nixpkgs.
+
+    mkdir -p native/linux/x64/
+    cp ${bisq-tor} ./tor
+    ${gnutar}/bin/tar -cJf native/linux/x64/tor.tar.xz tor
+    ${zip}/bin/zip -r opt/bisq/lib/app/desktop-${version}-all.jar native
```

```suggestion
    tar -cJf native/linux/x64/tor.tar.xz tor
    zip -r opt/bisq/lib/app/desktop-${version}-all.jar native
```

These are already in `nativeBuildInputs`, so don't need to be referenced again.

Direct references like these don't work when cross compiling. That doesn't apply directly here, but it's a good habit.

---

Same applies for imagemagick below.